src/cmd/projectImport.go

```diff
@@ -50,16 +58,17 @@ func projectImportCmd() *cmdBuilder.Cmd {
 				},
 			)
 			if err != nil {
+				uxBlocks.PrintError(styles.ErrorLine(i18n.T(i18n.ProjectImportFailed)))
 				return err
 			}
 
-			responseOutput, err := importProjectResponse.Output()
+			projectOutput, err := importProjectResponse.Output()
 			if err != nil {
-				return err
+				uxBlocks.PrintError(styles.ErrorLine(i18n.T(i18n.ProjectImportFailed)))
```

Comment:
you're only printing error here, but execution will continue, you need to return error

Reply:
Alright will look into this - thanks for pointing it out 😸